core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -127,12 +141,14 @@ public void draw(SpriteBatch batch, float runtime) {
 				}
 			}
 			/* Does the ball hit the edge? */
-			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN && Math.toDegrees(ball.getAngle()) > 90) {
+			if (ball.getX() - ball.getRadius() <= Config.BOUNCE_X_MIN
```

Comment:
Yes, the CheckStyle tool. I implemented a modified Google Java style, as I thought that they should probably have a nice style. With that style comes a line length restraint. It was 100 originally but I put it to 120 as I found it a better number for Java. In the future we could all sit down and discuss some of the style constrains (after the initial delivery)